test/powershell/engine/PropertyAccessor.Tests.ps1

```diff
@@ -0,0 +1,103 @@
+#
+# Functional tests to verify basic conditions for IO to the PowerShellProperties.json files
+# The properties files are supported on non-Windows OSes, but the tests are specific to 
+# Windows so that file IO can be verified using supported cmdlets.
+# 
+
+try {
+    # Skip these tests when run against "InBox" PowerShell
+    $IsInbox = $PSHOME.EndsWith('\WindowsPowerShell\v1.0', [System.StringComparison]::OrdinalIgnoreCase)
+    $productName = "PowerShell"
+
+    #skip all tests on non-windows platform
+    $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+    $PSDefaultParameterValues["it:skip"] = (!$IsWindows -or $IsInbox)
```

Comment:
Thanks for take Nano/IoT PS into account. Now it makes sense to have `$skipTest = !$IsWindows -or $IsInbox`, because the `if ($IsWindows)` below should actually be `if (!skipTest)`.